loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -23,10 +23,12 @@
 public class FMLConfig
 {
     public enum ConfigValue {
+        DISABLE_CONFIG_WATCHER("disableConfigWatcher", Boolean.FALSE, "Disables File Watcher. Used to automatically update config if its file has been modified."),
```

Comment:
What is the point of disabling the config watcher? If it is to fix a bug, it might already be fixed in upstream NightConfig. Or at least it was reported to TheElectronWill some time ago and he said a fix was in the pipeline?

Reply:
Basically, there's no point as such, but I'd like to be able to do so

Reply:
FWIW, I'm in favor of this change - there are outstanding issues with the watcher on our end that are not addressed, such as the `Loading` & `Reloading` events being able to fire simultaneously, and the whole concept being quite broken with editors that don't save atomically. So I think giving users the option to disable the watcher is a good stopgap until someone has time to properly rewrite that logic.

Reply:
Should we maybe disable it by default then for now? (at least in production)

Reply:
How about a complete remove?

Reply:
No - it would be nice to fix it eventually so please leave it in.

Reply:
Ok, what else do I need to do to get this pr merged? (I can't continue working on one mod without these changes)